docs/en/serverless/quickstarts/monitor-hosts-with-elastic-agent.mdx

```diff
@@ -0,0 +1,119 @@
+
+1. <DocLink slug="/serverless/observability/create-an-observability-project">Create a new ((observability)) project</DocLink>, or open an existing one.
+1. In your ((observability)) project, go to **Add Data**.
+1. Select **Collect and analyze logs**, and then select **Auto-detect logs and metrics**.
+1. Copy the command that's shown. For example:
+  ![Quick start showing command for running auto-detection](../images/quickstart-autodetection-command.png)
+  You'll run this command to download the auto-detection script and scan your system for observability data.
+1. Open a terminal on the host you want to scan, and run the command.
+1. Review the list of log files:
+    - Enter `Y` to ingest all the log files listed
+    - Enter `n` to either exclude log files or specify additional log paths. Enter `Y` to confirm your selections.
+
+  When the script is done, you'll see a message like "((agent)) is configured and running."
+
+There might be a slight delay before logs and other data are ingested.
+
+<DocCallOut title="Need to scan your host again?">
+  You can re-run the script on the same host to detect additional logs.
+  The script will scan the host and reconfigure ((agent)) with any additional logs that are found.
+  If the script misses any custom logs, you can add them manually by entering `n` after the script has finished scanning the host.
+</DocCallOut>
+
+## Visualize your data
+
+After installation is complete and all relevant data is flowing into Elastic,
+the **Visualize your data** section will show links to assets you can use to analyze your data.
+Depending on what type of observability data was collected,
+the page may link to assets for the following integrations:
+
+{/* TODO: Add more detail to this section to highlight "the salient points of the specific dashboards or pages." Waiting for info from SMEs. */}
+
+<DocTable columns={[{ title: "Integration assets" }, { title: "What insights can you get?" }]}>
+  <DocRow>
+    <DocCell>**System**</DocCell>
+    <DocCell>
+      - Identify high disk, memory, or CPU usage
+      - Monitor throughput
+      - Detect log spikes and drops
+    </DocCell>
+  </DocRow>
+  <DocRow>
+    <DocCell>**Apache**</DocCell>
+    <DocCell>ADD DESCRIPTION HERE</DocCell>
+  </DocRow>
+  <DocRow>
+    <DocCell>**Docker**</DocCell>
+    <DocCell>ADD DESCRIPTION HERE</DocCell>
+  </DocRow>
+  <DocRow>
+    <DocCell>**Nginx**</DocCell>
+    <DocCell>ADD DESCRIPTION HERE</DocCell>
+  </DocRow>
+  <DocRow>
+    <DocCell>**Custom .log files**</DocCell>
+    <DocCell>ADD DESCRIPTION HERE</DocCell>
+  </DocRow>
+</DocTable>
+
+For example, you can navigate the **Host overview** dashboard to explore detailed metrics about system usage and throughput.
+Metrics that indicate a possible problem are highlighted in red.
+
+{/* TODO: Decide where to crop this dashboard...it's kind of big to include it its entirety. */}
+
+![Host overview dashboard](../images/quickstart-host-overview.png)
```

Comment:
@akhileshpok Take a look at the screenshot in the preview and let me know if/where you want me to crop it.

Reply:
Nevermind...I will just crop to a reasonable size. If I've removed too much info, just let me know.